### DISCUSSION TITLE
how do i update the lazyvim itself?

### DISCUSSION THREAD

**Discussion by raceychan** [LazyVim/LazyVim#1201] [Q&A] [answered]
i have been looking for a solution for this, lazyvim itself is under rapid development, and i don't find instruction on the docs about how to update lazyvim itself.
I installed lazyvim via gitclone

**Comment by dpetka2001** [comment, marked as answer]
Lazyvim is itself a plugin like all others. So when an update is available, it will be shown in `Lazy`. Then you just press `U` to update all available plugins (Lazyvim included).
